<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoCalibrationDialog</class>
 <widget class="QDialog" name="VideoCalibrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Measurement Calibration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1f232a;</string>
  </property>
  <widget class="QLabel" name="videoLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>711</width>
     <height>561</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="screenshotBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: #1f232a;</string>
   </property>
   <property name="text">
    <string>Capture Screenshot</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/image.svg</normaloff>:/icons/icons/image.svg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="lineMeasurementBtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Line Measurement</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/plus-circle.svg</normaloff>:/icons/icons/plus-circle.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="importPictureBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Import Picture</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/folder-plus.svg</normaloff>:/icons/icons/folder-plus.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="nextBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>680</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/arrow-right.svg</normaloff>:/icons/icons/arrow-right.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
